templates/deployment.yaml

```
@@ -107,9 +107,9 @@ spec:
         {{- end }}
         readinessProbe:
           httpGet:
-            path: /_health_check
+            path: {{ default /_health_check .values.tfe.readinessProbePath }}
```

There are a few edits are needed for tis to function as intended
```suggestion
            path: {{ default "/_health_check" .Values.tfe.readinessProbePath }}
```

---

Change made as requested